apps/src/lib/levelbuilder/CourseVersionPublishingEditor.jsx

```diff
@@ -264,6 +266,15 @@ export default class CourseVersionPublishingEditor extends Component {
             />
           </label>
         )}
+        {this.props.courseOfferingEditorLink && (
+          <Button
+            __useDeprecatedTag
+            color={Button.ButtonColor.gray}
+            href={this.props.courseOfferingEditorLink}
+            target="_blank"
+            text={'Edit Course Offering'}
+          />
+        )}
```

Comment:
I don't see any other language around "course offering" on this page. I think it would be helpful to include a tooltip briefly describing what kinds of things can be edited by clicking this button.